_data/events.yml

```diff
@@ -38,7 +38,8 @@
   url: https://local.hackcu.org
   
 - name: HackCU IV
-  location: TBD
+  date: 2018-02-24 8:00 AM
+  location: SEEC Building (East Campus)
```

Comment:
Can we add a hyperlink to a map or something? That would make it easier for people who are not familiar with the building (like me!) to find it.

Reply:
Never mind, just realized that we have a link to the actual website where we will soon have the map of the venue